contracts/HybridCustody.cdc

```diff
@@ -880,90 +910,103 @@ pub contract HybridCustody {
 
         /// Checks the validity of the encapsulated account Capability
         ///
-        pub fun check(): Bool {
+        access(all) view fun check(): Bool {
             return self.acct.check()
         }
 
         /// Returns a reference to the encapsulated account object
         ///
-        pub fun borrowAccount(): &AuthAccount {
-            return self.acct.borrow()!
+        access(Owner) view fun borrowAccount(): auth(Storage, Contracts, Keys, Inbox, Capabilities) &Account {
```

Comment:
I have a question: here it marks `view` function

I am a bit confused, of course, it is used for pre/post conditions.

But what I want to ask is, will it affect the normal use of this AuthAccount Reference? 
Will writing operations be unable to be performed?

Reply:
From my understanding, `view` restricts only the scope of the function - mutating methods can call `view` methods, but `view` methods cannot call mutating methods. So even though the reference is retrieved within this scope as a read-only return, a caller could then execute state change on the returned reference.

Reply:
Got it, thank you!